store.rb

```diff
@@ -35,14 +59,25 @@ def session_id
   def stop()
     @log.warn "stopping the store"
     Process.kill("TERM", Process.pid) unless $EXIT
+    @root_watcher.unsubscribe unless @root_watcher.nil?
```

Comment:
it seems more popular (and shorter) to write it in this way: ` @root_watcher.unsubscribe if @root_watcher`